packages/docusaurus-plugin-content-blog/src/frontMatter.ts

```diff
@@ -1,83 +0,0 @@
-/**
```

Comment:
need to restore this whole file, and just import the LastUpdate front matter schema in it